package.json

```diff
@@ -31,8 +31,8 @@
     "lint-staged": "lint-staged",
     "spellcheck": "yaspeller-ci *.md",
     "test": "yarn run test:unit && yarn run test:acceptance && yarn run lint && yarn run spellcheck",
-    "test:acceptance": "karma start",
-    "test:acceptance:watch": "WATCH=true yarn run test:acceptance -- --no-single-run",
+    "test:acceptance": "NODE_ENV=production karma start",
```

Comment:
Should I? I think, this way tests will be more consistent on local machines, too.